package.json

```diff
@@ -39,6 +39,7 @@
     "joi": "^9.1.0",
     "joi-objectid": "^2.0.0",
     "joiql": "0.1.0",
+    "jquery-on-infinite-scroll": "^1.0.1",
```

Comment:
Oops also it looks like we saved this package twice. feel free to remove this one